### DISCUSSION TITLE
第6节课作业(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#469] [作业]
第6节课作业(13班)

**Comment by qingyin741** [comment]
![Snipaste_2024-01-22_17-04-00](https://github.com/InternLM/tutorial/assets/118804949/2f25b9ac-bd94-4232-b1c9-89417c984cfa)

![Snipaste_2024-01-22_17-04-33](https://github.com/InternLM/tutorial/assets/118804949/70e116f2-4b74-403e-a6fa-26787e928154)

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/FQ1GwPEIhigehFkyY0Nc0bv8nxb?from=from_copylink

**Comment by qingtao12138** [comment]
第六课作业：
https://docs.qq.com/doc/DRE1oclBPWkdlSEZa

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135857262
